src/gui_helper.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <QMessageBox>
+#include <QString>
+
+
+inline void showErrorDialog(const QString& message) {
+	QMessageBox msgBox(QMessageBox::Icon::Critical, QString("GUI ERROR"), message);
+	msgBox.exec();
```

Comment:
Windows: it does not set focus on OK button. One can not just press ENTER to quit the error dialog.
@r0ost3r Can you check if it is the same behaviour on Linux?

Reply:
It is not about the focus, but `grabKeyboard()` blocks **ENTER** for the button.
We need some `grab` - ON - OFF logic.

Reply:
moved to #78